pkg/cmd/create/create_cluster.go

```diff
@@ -23,10 +19,9 @@ type KubernetesProvider string
 // CreateClusterOptions the flags for running create cluster
 type CreateClusterOptions struct {
 	options.CreateOptions
-	InstallOptions   InstallOptions
-	Flags            initcmd.InitFlags
-	Provider         string
-	SkipInstallation bool `mapstructure:"skip-installation"`
+	InstallOptions InstallOptions
```

Comment:
I don't think there is a reason to keep the other 2 flags (InstallOptions, init Flags) since we are removing the ability to call install from the `create cluster` command. I would also remove them.